File: php/testcode/benchmark_test_00807.php
<?php
require_once __DIR__ . '/shared.php';

class UserModel042 {
    private string $name = '';
    private string $email = '';
    public function setName(string $name): void { $this->name = $name; }
    public function setEmail(string $email): void { $this->email = $email; }
}

function benchmarkTest00807(BenchmarkRequest $req): BenchmarkResponse {
    $model = new UserModel042();
    $model->setName($_POST['name'] ?? '');
    $model->setEmail($_POST['email'] ?? '');
    return BenchmarkResponse::ok('set');
}

File: rust/testcode/benchmark_test_00807.rs
pub fn handle(req: &super::shared::BenchmarkRequest) -> super::shared::BenchmarkResponse {
    let user_path = req.param("dir");

    match std::fs::create_dir_all(&user_path) {
        Ok(_) => super::shared::BenchmarkResponse::ok("Directory created"),
        Err(e) => super::shared::BenchmarkResponse::error(&e.to_string()),
    }
}
